projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -59,18 +87,13 @@ frontend-shared-components-release:
   before_script:
     - cd projects/frontend/shared-components/gui
   script:
-    - ../../cicd/build_shared.sh
     - ../../cicd/publish_shared.sh $CI_PIPELINE_ID $NPM_REGISTRY $NPM_TOKEN
```

Comment:
Where does the package we publish to npm registry come from if we don't run the build script?